src/uwtools/rocoto.py

```diff
@@ -26,6 +31,57 @@ def _add_jobname(tree: dict) -> None:
 
 
 # Public functions
+def realize_rocoto_xml(
+    input_yaml: str,
+    input_template: str,
+    rendered_output: str,
+    schema_file: str,
+) -> None:  # pragma: no cover
```

Comment:
The other "realize" function return a bool. Perhaps we should keep this one consistent with those?

Reply:
Ah yes, thanks!